tensorboard/webapp/metrics/views/card_renderer/utils_test.ts

```diff
@@ -243,4 +248,62 @@ describe('metrics card_renderer utils test', () => {
       });
     });
   });
+
+  describe('#maybeSetClosestStartStep', () => {
+    it('sets startStep to closest step', () => {
+      const viewSelectedTime = {
+        startStep: 0,
+        endStep: null,
+        clipped: false,
+      };
+
+      expect(maybeSetClosestStartStep(viewSelectedTime, [10, 20, 30])).toEqual({
+        startStep: 10,
+        endStep: null,
+        clipped: false,
+      });
+    });
+
+    it('does not set startStep when closest step is null', () => {
```

Comment:
optional nit: I would rather this be called 'does not set startStep when passed an empty array of steps'

Reply:
make sense to me. thanks!